src/test/java/com/easypost/ShipmentTest.java

```diff
@@ -0,0 +1,224 @@
+package com.easypost;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Shipment;
+import com.easypost.model.Rate;
+import com.easypost.model.ShipmentCollection;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ShipmentTest {
+    private static Shipment shipment;
+
+    /**
+     * Setup the testing environment for this file.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @BeforeClass
+    public static void setup() throws EasyPostException {
+        EasyPost.apiKey = "EASYPOST_TEST_API_KEY";
+
+        shipment = Shipment.create(Fixture.fullShipment());
+    }
+
+    /**
+     * Test creating a shipment.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @Test
+    public void testCreate() throws EasyPostException {
+        Shipment shipment = Shipment.create(Fixture.fullShipment());
+
+        assertTrue(shipment instanceof Shipment);
+        assertTrue(shipment.getId().startsWith("shp_"));
+        assertEquals("PNG", shipment.getOptions().get("label_format"));
+        assertEquals(123.0, shipment.getOptions().get("invoice_number"));
```

Comment:
This shouldn't be right. `123` does not equal `123.0`.

Reply:
getOptions() returns a Hashmap, the value for "invoice_number" key is double which is 123.0

Reply:
That's definitely wrong then. The type of this field is a string. Please correct it from `double` to `string` and add a line entry to the CHANGELOG for the `NEXT RELEASE` saying this was corrected.

Reply:
**`invoice_number`** does not belong to any class attribute, apparently, it was because the test input is double, I have changed the type in Fixture from `double` to `string`.